src/outfit.cpp

```diff
@@ -40,31 +40,34 @@ bool Outfits::loadFromXml()
 			continue;
 		}
 
-		outfits[type].emplace_back(
-		    outfitNode.attribute("name").as_string(), pugi::cast<uint16_t>(lookTypeAttribute.value()),
-		    outfitNode.attribute("premium").as_bool(), outfitNode.attribute("unlocked").as_bool(true));
+		const uint16_t lookType = pugi::cast<uint16_t>(lookTypeAttribute.value());
```

Comment:
Since we no longer have two maps it is understandable to add a new value for each outfit to store the gender of said outfit.

<details>

<summary>Outfit</summary>

```hpp
struct Outfit
{
	Outfit(std::string_view name, uint16_t lookType, PlayerSex_t sex, bool premium, bool unlocked) :
	    name{name}, lookType{lookType}, sex{sex}, premium{premium}, unlocked{unlocked}
	{}

	bool operator==(const Outfit& otherOutfit) const
	{
		return name == otherOutfit.name && lookType == otherOutfit.lookType && sex == otherOutfit.sex &&
		       premium == otherOutfit.premium && unlocked == otherOutfit.unlocked;
	}

	std::string name;
	uint16_t lookType;
	PlayerSex_t sex;
	bool premium;
	bool unlocked;
};
```

</details>

And to emplace the values:
```cpp
const auto lookType = pugi::cast<uint16_t>(lookTypeAttribute.value());
outfits.emplace(std::piecewise_construct, std::forward_as_tuple(lookType),
		                std::forward_as_tuple(outfitNode.attribute("name").as_string(), lookType,
		                                      static_cast<PlayerSex_t>(type), outfitNode.attribute("premium").as_bool(),
		                                      outfitNode.attribute("unlocked").as_bool(true)));
```

Remember use cclang if u copy this code.